### ISSUE TITLE
[Feat]: Populate UpdateNodeInfo message with services

### ISSUE THREAD

**Issue by MrZammler** [netdata/netdata#12876]
### Problem

Currently the `services` field (https://github.com/netdata/netdata/blob/7a6a36d2dfd708cacf620610118e307c04890bce/database/sqlite/sqlite_aclk_node.c#L55) in `sql_build_node_info` is NULL. We need to fill this with the unique combinations of `modules` and `plugins`, so that from this the cloud can extract and display services related to that agent. 

This functionality existed in the old architecture.

### Description

Currently the cloud does not receive any services list.

### Importance

must have

### Value proposition

1. Provide a list of services on the cloud.

**Comment by vkuznecovas** [comment]
Just a thought - since this is already not working, we don't have to use the `services` property in the `UpdateNodeInfo`, we can create one that better corresponds to what we're actually sending there - maybe `collectors`?

**Comment by MrZammler** [comment]
Hey Viktoras, yes, we could but we have the name already -> https://github.com/netdata/aclk-schemas/blob/d8342ee6d932c152a78c9fe886281fe28170a6c4/proto/nodeinstance/info/v1/info.proto#L105 changing it I think we would need to keep it forever and it's id...

**Comment by vkuznecovas** [comment]
I agree, but my point is since we're not using this field we can comment it out and add a new one to contain the items that it actually will contain, as we're not going to be receiving services from the agent side, but the collectors, right?

**Comment by MrZammler** [comment]
This is discussed, there is code to test for the functionality (it will be completely new message), and will be tested and PR next week with @vkuznecovas .

We'll keep it open, and close it when it works.

**Comment by MrZammler** [comment]
Closing in favor of #13330 .
